README.md

```diff
@@ -123,21 +136,153 @@ For a more detailed description of NNCF usage in your training code, see [this t
 For in-depth examples of NNCF integration, browse the [sample scripts](#model-compression-samples) code, or the [example patches](#third-party-repository-integration) to third-party repositories.
 For FAQ, visit this [link](./docs/FAQ.md).
 
+### Usage examples of Post-Training Quantization
+
+NNCF provides [samples](#post-training-quantization) that demonstrate Post-Training Quantization usage for PyTorch, TensorFlow, ONNX, OpenVINO.
```

Comment:
```suggestion
NNCF provides [samples](#post-training-quantization), which demonstrate Post-Training Quantization usage for PyTorch, TensorFlow, ONNX, OpenVINO.
```

Reply:
I'm afraid the previous version is correct